tests/test_viewsets.py

```diff
@@ -187,6 +188,22 @@ def test_validates_username(self):
                 response.rendered_content.decode("utf-8"),
             )
 
+            # Test error returned when username doesn't match regex
+            mock_func.return_value = {
+                "given_name": "jane",
+                "family_name": "doe",
+                "email": "jane@doe.com",
+                "preferred_username": "12345",
+            }
+            data = {"id_token": "ssad9012.fdfdfdswg4gdfs.sadadsods"}
```

Comment:
The id_token seems to be used acrosee multiple tests, can we have this stored in a settings file?

Reply:
I don't think this should be in settings... It's just a random mocked value... that doesn't particular do much even if it's changed